test/framework/kogitoapp.go

```diff
@@ -105,7 +109,15 @@ func crDeployService(namespace string, kogitoAppDeployment KogitoAppDeployment)
 
 	setupBuildImageStreams(kogitoApp)
 
-	if kogitoAppDeployment.Native {
+	if len(kogitoAppDeployment.Requests) > 0 || len(kogitoAppDeployment.Limits) > 0 {
+		kogitoApp.Spec.Resources = ToResourceRequirements(kogitoAppDeployment.Requests, kogitoAppDeployment.Limits)
+	}
+
+	if len(kogitoAppDeployment.BuildRequests) > 0 || len(kogitoAppDeployment.BuildLimits) > 0 {
+		kogitoApp.Spec.Build.Resources = ToResourceRequirements(kogitoAppDeployment.BuildRequests, kogitoAppDeployment.BuildLimits)
+	}
+
+	if kogitoAppDeployment.Native && &kogitoApp.Spec.Build.Resources == nil {
```

Comment:
This nil check is not effective. In case the resources are not specified then default value is used. If the property is not a pointer then it cannot be nil.

Reply:
Ok, got it. Will change it.